src/preact/component/iframe.js

```diff
@@ -60,7 +60,6 @@ export function IframeEmbedWithRef(
     onReadyState,
     sandbox,
     src,
-    ...rest
```

Comment:
Shouldn't we still pass `...rest` to iframe? Note also that the `iframeStyle` prop can be used as plain `style` with this change.